### ISSUE TITLE
Feature request: No installer for the Mac build

### ISSUE THREAD

**Issue by snap9** [cryptocat/cryptocat#131]
The Cryptocat Mac installer requires the admin password. This is worrying for security-conscious Mac users.

Please turn the Mac app into a simple .app. No installer and no admin password requirement.

**Comment by kaepora** [comment]
Nope, sorry. You can right click the package and select "Show Package Contents" in order to extract the package manually, but the package will remain a package for two reasons:

1. Ease of installation,
2. OS X Package signing,
2. Apple guidelines for non-MAS apps specifically recommend packages.

**Comment by snap9** [comment]
@kaepora 
- The vast majority of non-MAS Mac apps that I have seen (been a Mac user for +10 years) come as .app within either a .zip file or a .dmg bundle. Very few have installers.
- .app can also be code signed just fine with a Developer ID.
- If you really prefer packages, can you at least make it not require an admin password? Spotify for instance is one of the few apps that have an installer, but at least Spotify doesn't require an admin password to install.

**Comment by kaepora** [comment]
I am reopening this issue until I investigate it sufficiently to reply to your latest comment.

**Comment by Eitot** [comment]
I agree with snap9. It surprised me to see an installer package. They can be abused easily and are therefore met with ample justified skepticism. An installer package is useful if you need to place helper files and tools elsewhere on the system, particularly in areas where standard users have no write access. Given that this is just a single app bundle that needs no write access beyond the home directory, using an installer package is really an unnecessary barrier.

I have not come across an official recommendation for either method, but Apple themselves distributes software in both ways. Even large suites like Xcode:
![screen shot 2](https://cloud.githubusercontent.com/assets/15073177/15455895/2e46a55e-2061-11e6-9408-65401610aafb.png)

They have a guide for signing app bundles here:
https://developer.apple.com/library/mac/documentation/IDEs/Conceptual/AppDistributionGuide/DistributingApplicationsOutside/DistributingApplicationsOutside.html#//apple_ref/doc/uid/TP40012582-CH12-SW2

**Comment by Eitot** [comment]
Just to add: Apple is making some changes to Gatekeeper in macOS Sierra. When you distribute an application in a disk image or archive, then you also need to sign that container with the same certificate to avoid the application from being translocated at runtime. Apple still fully supports both methods – signed app bundle and signed installer package – and makes no mention of which one is preferred for which situation.

There is some more information about this as well as the changes specific to Sierra:
https://developer.apple.com/library/prerelease/content/technotes/tn2206/_index.html#//apple_ref/doc/uid/DTS40007919-CH1-TNTAG17

**Comment by kaepora** [comment]
@Eitot Great post, thanks for the heads-up.
